pkg/operator/ceph/pool/status.go

```diff
@@ -46,12 +44,24 @@ func updateStatus(ctx context.Context, client client.Client, poolName types.Name
 		pool.Status = &cephv1.CephBlockPoolStatus{}
 	}
 
+	// add pool ID to the status
```

Comment:
nit: How about moving this new pool id detection to a helper method, similar to `updateStatusInfo()`? Then it keeps the updateStatus method simpler.